app/controllers/testcontroller.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+class AdminController extends \BaseController {
+	
+	/**
+	 * Sends beta invite email
+	 */
+	public function invite(){
+
+        return "Do nothing. no invites sent";
+
+		// Get the emails to invite
+//		$users 		=	Beta::whereStatus(0)->lists('email');
+//		// At this email always for testing
+//		$users[]	=	"jefrycruz88@gmail.com";
+
+		// Set the all statuses to 0
+//		Beta::whereStatus(0)->update(array('status' => 1));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.109:9000/coding_rules#rule_key=php%3AS125)